app/platform/fabric/gateway/FabricGateway.ts

```diff
@@ -488,4 +488,18 @@ export class FabricGateway {
 
 		return null;
 	}
+
+	async getActiveOrderersList(channel_name) {
+		const network = await this.gateway.getNetwork(channel_name);
+		let orderers = [];
+		let ordererAtrributes;
```

Comment:
[nitpick]: this can be local variable

Reply:
@adityajoshi12 ,
Changed the code as suggested.

Reply:
@adityajoshi12 ,
changed the code as suggested.